lib/template.js

```diff
@@ -45,7 +45,9 @@ JsonTemplate.prototype.compile = function() {
     function parse(item) {
       var matched = false;
       var templates = /\{([^\{\}]+)\}/g;
+      /* eslint-disable no-cond-assign */
       while (match = templates.exec(item)) {
+        /* eslint-enable no-cond-assign */
```

Comment:
for now, but we should refac later to

```
var match;
while (match) {
  match = templates.exec(item);
  ...
}
```

Not so important ATM because we can fix outside this giant PR and just backport the fix.